consutil/lib.py

```diff
@@ -75,16 +80,32 @@ def getBusyDevices():
             busyDevices[linenum_key] = (pid, date)
     return busyDevices
 
-# returns baud rate of device corresponding to line number
+# returns baud rate and flow control settings of device corresponding to line number
 # input: linenum (str)
-def getBaud(linenum):
-    checkDevice(linenum)
-    cmd = "sudo stty -F " + DEVICE_PREFIX + str(linenum)
-    output = run_command(cmd)
+def getConnectionInfo(linenum):
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    entry = config_db.get_entry(TABLE_NAME, str(linenum)) 
```

Comment:
is it possible that we cannot get an entry here?

Reply:
get_entry() will return an empty dictionary if the table or entry doesn't exist.